latest_cov_build.py

```diff
@@ -0,0 +1,26 @@
+import tarfile
+import sys
+import os
+import taskcluster
+try:
+    from urllib.parse import urlencode
+    from urllib.request import urlopen, urlretrieve
+except ImportError:
+    from urllib import urlencode, urlretrieve
+    from urllib2 import urlopen
+
+
+index = taskcluster.Index()
+queue = taskcluster.Queue()
+route = "gecko.v2.mozilla-central.latest.firefox.linux64-ccov-opt"
+taskId = index.findTask(route)['taskId']
+artifactName = 'public/build/target.tar.bz2'
+name = os.path.join('ccov-artifacts', taskId +
+                    'artifacts.public.build.target.tar.bz2')
+url = queue.buildUrl('getLatestArtifact', taskId, artifactName)
```

Comment:
Have you tried `queue.getLatestArtifact`?

Reply:
There might even be a way to retrieve the artifact directly without getting the task first: https://github.com/taskcluster/taskcluster-client.py#get-artifact-from-indexed-task.

Reply:
Thanks for previous comments. :) 
No, I haven't tried because as I have understood from Taskcluster API it was written: `API Clients, this method will redirect you to the artifact, if it is stored externally. Either way, the response may not be JSON. So API client users might want to generate a signed URL for this end-point and use that URL with a normal HTTP client.` - so, I've decided that it I will anyway need to build URL.
But I can try to redo it with this method if it is better. :)

Reply:
Yes, it would be better as it is more clear

Reply:
I wanted to ask here:
 1) I have tried this method https://github.com/taskcluster/taskcluster-client.py#get-artifact-from-indexed-task, but it hasn't worked. When I've tried to use the link constructed by pattern described in the documentation of this method, it returns Error. Here it is: https://index.taskcluster.net/task/gecko.v2.mozilla-central.latest.firefox.linux64-ccov-opt/artifacts/public/target.tar.bz2
2) queue.getLatestArtifact() method is void, and I have seen in documentation (honestly, for Javascript) that they use queue.buildUrl('getLatestArtifact', taskId, artifactName) method for getting url.

Reply:
1 is strange, the URL should be `https://index.taskcluster.net/v1/task/gecko.v2.mozilla-central.latest.firefox.linux64-ccov-opt/artifacts/public/build/target.tar.bz2`. Have you tried `index.findArtifactFromTask('gecko.v2.mozilla-central.latest.firefox.linux64-ccov-opt', 'public/build/target.tar.bz2')`? Or you can do `urlretrieve('https://index.taskcluster.net/v1/task/gecko.v2.mozilla-central.latest.firefox.linux64-ccov-opt/artifacts/public/build/target.tar.bz2')` directly
2 yes they use that behind the scenes, but it could change in the future so it's better to use their interface rather than reimplementing it.